fmanimashaun/myblog
# Blog app - Views

# Description

This pull request introduces a new view page to our application. The view page is designed to display user-specific data in a user-friendly format.

## Changes Made

1. Created a new view file in the `views/users`  and `views/posts` directory.
2. Added HTML and embedded Ruby code to display user data.
3. Updated the `UsersController` to pass the necessary data to the view.

## How to Test

To test the new view page, navigate to the appropriate URL in your local development environment. The page should load without errors and display the correct user data.

## Screenshots

![image](https://github.com/fmanimashaun/myblog/assets/22054464/8bbc29d0-65fe-4133-90c8-6d7d123cf7cb)
![image](https://github.com/fmanimashaun/myblog/assets/22054464/71a6a3c9-9619-4985-9b3a-3341e1ddb346)
![image](https://github.com/fmanimashaun/myblog/assets/22054464/4bec063e-8a64-4561-af84-c7eabe80ff9c)
![image](https://github.com/fmanimashaun/myblog/assets/22054464/8b237a6f-5961-465c-ac02-8878bea18703)


## Checklist

- [x] I have followed the coding standards and style of the project
